lib/internal/modules/cjs/loader.js

```diff
@@ -189,6 +189,9 @@ let { startTimer, endTimer } = debugWithTimer('module_timer', (start, end) => {
   endTimer = end;
 });
 
+const { tracingChannel } = require('diagnostics_channel');
+const onRequire = getLazy(() => tracingChannel('module.require'));
```

Comment:
getLazy is required otherwise it will cause segmentation fault on nodejs build. I haven't figured it out in-depth, but I suspect some module required by `dc` isn't available before Node.js fully startup.

Reply:
My guess would be that [`WeakReference`](https://github.com/nodejs/node/blob/main/lib/diagnostics_channel.js#L43) is not initialized yet.